app/views/application/home.html.slim

```diff
@@ -0,0 +1,3 @@
+div class="text-center mt-5" 
+  h2 Welcome to 
+  h1 Ruby Google Scraper
```

Comment:
Should we use I18n here 🤔

Reply:
Fixed in 4f0475c9f509e557881f7842bf7f1787b26230ab